package NKikimrTxCoordinator;

// Request from new coordinator generation to the last known state actor
message TEvCoordinatorStateRequest {
    // Generation of the new tablet
    optional uint32 Generation = 1;
}

// Response with the last known state
message TEvCoordinatorStateResponse {
    // The last step that was potentially sent to mediators
    optional uint64 LastSentStep = 1;

    // The last step that was acquired by readers
    optional uint64 LastAcquiredStep = 2;

    // The last step that was confirmed to planners
    optional uint64 LastConfirmedStep = 3;
}
